tap_csv/tap.py

```diff
@@ -42,6 +42,7 @@ class TapCSV(Tap):
             th.StringType,
             description="A path to the JSON file holding an array of file settings.",
         ),
+        th.Property("add_metadata_columns", th.BooleanType, required=False, default=False),
```

Comment:
```suggestion
        th.Property(
            "add_metadata_columns",
            th.BooleanType,
            required=False,
            default=False,
            description="When True, add the metadata columns (`_sdc_source_file`, `_sdc_source_file_mtime`, `_sdc_source_lineno`) to output.",
        ),
```

Reply:
@TyShkan I added a suggestion for putting the description from the README in here. We scrape this data for MeltanoHub so its helpful to have in there.